### PR TITLE
Add two functional Texas State Technical College scrapers

### PR DESCRIPTION
**Pull request by HarrisonGregg**

### CODE REVIEW

File: osp_scraper/spiders/tstc.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                        'require_files': True
+                    },
+                    callback=self.parse_for_files
+                )
+
+    def extract_links(self, response):
+        for row in response.css("#tblSchedule tr")[2:]:
+            url = response.css("a[title='Course Syllabus']::attr(href)").get()
+            if url:
+                anchor = " ".join(row.css("td")[:2].css("::text").getall())
```
Comment (jodizzle) [inline osp_scraper/spiders/tstc.py]: It might be good to include the semester as part of the anchor.

File: osp_scraper/spiders/tstc.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+import scrapy
+
+from ..spiders.CustomSpider import CustomSpider
+
+class TSTCSpider(CustomSpider):
+    name = "tstc"
```
Comment (jodizzle) [inline osp_scraper/spiders/tstc.py]: There are a couple other campuses that are part of Texas State Technical College.  I believe they're all linked on this page: http://www.tstc.edu/syllabi/tstcsyllabi.  Are any other campuses going to need custom scrapers, and therefore unique names?

It also might be a good idea to name this scraper based on Harlingen regardless.

Comment (HarrisonGregg) [inline osp_scraper/spiders/tstc.py]: Oh cool. I didn't notice those.  I mapped one of them, added a scraper that works for two others, and renamed this one.

File: osp_scraper/spiders/tstc.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                callback=self.parse_for_courses
+            )
+
+    def parse_for_courses(self, response):
+        for row in response.css("tr"):
+            # Onclick attribute is of the form:
+            # opn('<url>','','scrollbars=yes,resizable=yes,width=1200,height=900')
+            onclick = row.css("a::attr(onclick)").get()
+            if onclick:
+                url = onclick.split("'")[1]
```
Comment (jodizzle) [inline osp_scraper/spiders/tstc.py]: Do you prefer `.split` to `.re_first` here for a particular reason?  I think either works, though.

**Pull request merged**